front/app/components/IdeaCards/EmptyIdeas.tsx

```diff
@@ -67,7 +67,7 @@ const EmptyIdeas = () => {
       <EmptyContainerInner>
         <IdeaIcon name="idea" />
         <EmptyMessage>
-          <EmptyMessageMainLine>
+          <EmptyMessageMainLine aria-live="polite">
```

Comment:
The aria-live should rather be on the IdeasList container div. We want to notice content changing inside the container, not on the components that get in and out.